File: webhook.py
from flask import Flask, request, Response

app = Flask(__name__)
VERIFY_TOKEN = 'prathamesh10'

@app.route('/webhook', methods=['GET'])
def verify_webhook():
    if (request.args.get('hub.mode') == 'subscribe' and
        request.args.get('hub.verify_token') == VERIFY_TOKEN):
        print('Webhook verified!')
        return request.args.get('hub.challenge'), 200
    return Response(status=403)

@app.route('/webhook', methods=['POST'])
def receive_webhook():
    data = request.get_json()
    print('Received:', data)
    return Response(status=200)

if __name__ == '__main__':
    app.run(port=3000)

File: api/webhook.js
//webhook.js
export default function handler(req, res) {
    const VERIFY_TOKEN = "prathamesh10"; // create your own secret token
  
    if (req.method === "GET") {
      const mode = req.query['hub.mode'];
      const token = req.query['hub.verify_token'];
      const challenge = req.query['hub.challenge'];
  
      if (mode && token) {
        if (mode === 'subscribe' && token === VERIFY_TOKEN) {
          console.log('WEBHOOK_VERIFIED');
          res.status(200).send(challenge);
        } else {
          res.sendStatus(403);
        }
      }
    } else if (req.method === "POST") {
      console.log('Received webhook event:', req.body);
      res.status(200).send('EVENT_RECEIVED');
    } else {
      res.sendStatus(404);
    }
  }
  